server/matchmaker/matchmaker_queue.py

```diff
@@ -40,13 +40,15 @@ def __init__(
         self,
         game_service: "GameService",
         name: str,
+        id: int,
```

Comment:
Do we worry about shadowing a builtin here? It's definitely fine for the attribute, but idk about function parameter